src/connman.service.in

```diff
@@ -11,6 +11,8 @@ Wants=network.target remote-fs-pre.target
 Type=dbus
 BusName=net.connman
 Restart=on-failure
+# The prestart modprobe for rfkill was added to fix a race condition where rfkill was not loaded yet.
```

Comment:
Although this fixes the problem i do not like it, a better way would be adding a modprobe file that makes sure the driver is probed in early boot stage.